docs/en/connector-v2/sink/Snowflake.md

```diff
@@ -0,0 +1,57 @@
+# Snowflake
+
+> Snowflake sink connector
+>
+> ## Description
+>
+> Write Snowflake data through [Jdbc connector](Jdbc.md).
+> Support Batch mode and Streaming mode. The tested Snowflake version is 3.xx
+> On the underlying implementation, through the jdbc driver of Snowflake, execute the upsert statement to write data to HBase.
+> Two ways of connecting Snowflake with Java JDBC. One is to connect to zookeeper through JDBC, and the other is to connect to queryserver through JDBC thin client.
+
+## Options
```

Comment:
Need a options table like this: https://seatunnel.apache.org/docs/2.3.1/connector-v2/sink/S3File#options